lib/lists.nix

```diff
@@ -6,8 +6,8 @@ rec {
 
   inherit (builtins) head tail length isList elemAt concatLists filter elem genList;
 
-  /*  Create a list consisting of a single element.  `singleton x' is
-      sometimes more convenient with respect to indentation than `[x]'
+  /*  Create a list consisting of a single element.  `singleton x` is
```

Comment:
I remember @edolstra insisting on that notation for quoting. It's not supposed to be markdown I think (applies also to the other changes in this commit).